var/spack/repos/builtin/packages/nalu-wind/package.py

```diff
@@ -38,6 +38,9 @@ class NaluWind(CMakePackage):
     variant('fftw', default=False,
             description='Compile with FFTW support')
 
+    variant('wind-utils', default=True,
```

Comment:
I would prefer the default for this to be false. There are a lot of different types of builds/uses and wind-utils is not needed for all of them.

Reply:
I changed the default to False.